application/app/src/main/java/com/example/buber/Services/ApplicationService.java

```diff
@@ -0,0 +1,5 @@
+package com.example.buber.Services;
+
+public class ApplicationService {
+    // TODO: Lukes code go here
```

Comment:
@LukeDelaCruz Hey java wouldn't compile unless I have some class in the service package, so I made a base application service class but you are totally welcome to change it to however you want.

Reply:
Thanks. Mind fixing this build error? Not really sure what your intention is to override `getContactInfo` on Driver.

![image](https://user-images.githubusercontent.com/32045517/74868547-aa028f80-5313-11ea-81cc-8eee39798ef7.png)

Reply:
Yup